Running ogip_check with
    input = inputs/unrec2.fits
    otype = None
    logfile = out/inputs3.logs/./unrec2.fits.check.log
    verbosity = 2
    dtype = none
    vonly = False
    meta_key = rxte
Attempting to open file inputs/unrec2.fits
File opened successfully.
Running basic FITS verification with external call to ftverify.
Nothing from ftverify, trying with astropy.io.fits.verify(option='exception')
